.github/workflows/build-debian-multiarch.yml

```
@@ -78,7 +78,7 @@ jobs:
           apt-get update --fix-missing
           apt-get upgrade -y
           apt-get install libsdl2-dev libsdl2-image-dev libsdl2-mixer-dev libsdl2-ttf-dev libfreetype6-dev libportmidi-dev libjpeg-dev fontconfig -y
-          apt-get install python3-setuptools python3-dev python3-pip python3-wheel python3-sphinx -y
+          apt-get install python3-setuptools python3-dev python3-pip python3-wheel python3-sphinx=7.1.2 -y
```

```suggestion
          apt-get install python3-setuptools python3-dev python3-pip python3-wheel python3-sphinx=="7.1.2" -y
```
not sure, why is it here different?

---

: Version '7.1.2' for 'python3-sphinx' was not found

---

ah, maybe this has to be pip installed instead of using apt-get

---

No, turns out that not all python versions we support support that version of sphinx. Looking now at solutions